dl4j-labs/src/main/java/ai/certifai/training/classification/MulticlassExcercise.java

```diff
@@ -0,0 +1,188 @@
+import org.nd4j.linalg.activations.Activation;
+import org.nd4j.linalg.dataset.DataSet;
+import org.nd4j.linalg.dataset.SplitTestAndTrain;
+import org.nd4j.linalg.dataset.ViewIterator;
+import org.nd4j.linalg.dataset.api.iterator.DataSetIterator;
+import org.nd4j.linalg.dataset.api.preprocessor.DataNormalization;
+import org.nd4j.linalg.dataset.api.preprocessor.NormalizerMinMaxScaler;
+import org.nd4j.linalg.io.ClassPathResource;
+import org.nd4j.linalg.learning.config.Nesterovs;
+import org.nd4j.linalg.lossfunctions.LossFunctions;
+
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+
+//Dataset
+//https://www.kaggle.com/zhangjuefei/birds-bones-and-living-habits
+public class MulticlassExcercise {
+    static int seed =123;
+    static int numInput=10;
+    static int numClass=6;
+    static int epoch=9000;
+    static int batch_size=64;
+    static double splitratio =0.7;
+    static double learningrate=1e-3;
+
+
+    public static void main(String[] args) throws Exception{
+
+        //set filepath
+        File datafile = new ClassPathResource("/birdclassify/bird.csv").getFile();
+
+        //File split
+        FileSplit fileSplit = new FileSplit(datafile);
+
+
+        //set CSV Record Reader and initialize it
+        RecordReader rr= new CSVRecordReader(1,',');
+        rr.initialize(fileSplit);
+
+        //Build Schema to prepare the data
+        Schema sc = new Schema.Builder()
+                .addColumnInteger("id")
+                .addColumnsFloat("huml","humw","ulnal","ulnaw","feml","femw","tibl","tibw","tarl","tarw")
+                .addColumnCategorical("type", Arrays.asList("SW","W","T", "R", "P", "SO"))
+                .build();
+
+        TransformProcess tp = new TransformProcess.Builder(sc)
+                .removeColumns("id")
+                .categoricalToInteger("type")
+                .build();
+
+//        Checking the schema
+        Schema outputschema=tp.getFinalSchema();
+        System.out.println(outputschema);
+
+
+        List<List<Writable>> alldata = new ArrayList<>();
+
+        while(rr.hasNext()){
+            alldata.add(rr.next());
+        }
+
+        List<List<Writable>> processdata =LocalTransformExecutor.execute(alldata, tp);
+
+        //Create iterator from process data
+        CollectionRecordReader collectionRR = new CollectionRecordReader(processdata);
+
+        //Input batch size , label index , and number of label
+        DataSetIterator dataSetIterator = new RecordReaderDataSetIterator(collectionRR, processdata.size(),10,6);
+
+//        //Create Iterator and shuffle the data
+        //Error due to zero data is obtain
+        DataSet fulldataset = dataSetIterator.next();
+        fulldataset.shuffle();
+//
+//        //Input split ratio
+        SplitTestAndTrain testAndTrain = fulldataset.splitTestAndTrain(splitratio);
+//
+          //Get train and test dataset
+        DataSet trainData =testAndTrain.getTrain();
```

Comment:
DataSet trainData = testAndTrain.getTrain();

space before and after the operator